azure-kusto-data/tests/test_telemetry.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+
+from azure.kusto.data._telemetry import KustoTracing, KustoTracingAttributes
+
+
+class TestTelemetry(unittest.TestCase):
```

Comment:
Needs more tests, especially for the async variants

Reply:
added - note we only implemented the use of a single function when async